### ISSUE TITLE
Disconnection messages while joining

### ISSUE THREAD

**Issue by YelehaUwU** [multitheftauto/mtasa-blue#3688]
### Describe the bug

Whenever getting kicked while loading, the client doesn't get the kick message, but instead gets a different one, like a timed out message. The serverconsole displays the kick message correctly, it's just the client displaying a timed out message.

### Steps to reproduce

1. Join a server
2. Get kicked while loading
3. The message pops up, displaying incorrect information.

### Version

Nightly server & client

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

**Comment by Fernando-A-Rocha** [comment]
100% confirmed issue. Has been happening for many versions.

This is unfortunate because the client won't be able to see the message displayed with kickPlayer or onPlayerConnect event cancel with msg.

**Comment by Fernando-A-Rocha** [comment]
If I implement a system for denying to access to my server under certain conditions (e.g. serial or ip whitelist), the player won't be able to see the reason message.

**Comment by ffsPLASMA** [comment]
yep having this issues for years now on ffs. they get a timed out message instead.

**Comment by SeeGy-Developer** [comment]
Able to reproduce
